README.md

```diff
@@ -1,4 +1,4 @@
-# eventstudyr
+# eventstudyr <img src='man/figures/logo.png' align="right" height="139" />
```

Comment:
Confirming I'm good with including the logo here. :-)

Reply:
Looks great! Would it be to hard to add badges like in [ggplot2](https://github.com/tidyverse/ggplot2). I'm thinking of the latest CRAN version and the number of downloads?

Reply:
@santiagohermo check out the [README](https://github.com/JMSLab/eventstudyr/tree/7d2e6086885751d84c77e87c57fa6c06b926f92d#eventstudyr-)!

As a side note, I think we should also include an image containing the outputted event study plot from running the code in the [example](https://github.com/JMSLab/eventstudyr/tree/7d2e6086885751d84c77e87c57fa6c06b926f92d#usage). What do you think?

Reply:
@nateschor just chiming in that I like this idea

Reply:
Thanks @nateschor! The README is looking very neat 👍 

To reply to this

> As a side note, I think we should also include an image containing the outputted event study plot from running the code in the [example](https://github.com/JMSLab/eventstudyr/tree/7d2e6086885751d84c77e87c57fa6c06b926f92d#usage). What do you think?

I think that the proper way to do this would be to create the README knitting an `.Rmd` file, which would automatically run that code. For instance, [this Rmd file](https://github.com/tidyverse/ggplot2/blob/main/README.Rmd) creates ggplot2's README. They seem to declare the location of the generated figure [there](https://github.com/tidyverse/ggplot2/blob/f7246d4ad9aeee46890f60e3821d762f390378ed/README.Rmd#L7-L13). 

I tried this approach in https://github.com/JMSLab/eventstudyr/pull/37/commits/22665e8103b3d7a0c48e1b51da9f75e8f02225a6. If you disagree we can always reverse that commit @nateschor!